doc/userguide/rules/tls-keywords.rst

```diff
@@ -3,6 +3,20 @@ SSL/TLS Keywords
 
 Suricata comes with several rule keywords to match on various properties of TLS/SSL handshake. Matches are string inclusion matches.
 
+tls.subject
+-----------
+
+Legacy keyword to match TLS/SSL certificate Subject field.
+
+example:
+
+
+::
+
+  tls.subject:"CN=*.googleusercontent.com"
+
+Case sensitive, can't use 'nocase'.
+
```

Comment:
The idea is that this becomes a subsection of the following `tls.cert_subject` section. That way, we bring attention to the current keyword, while still documenting the legacy keywords.

(The same applies to the section for the other legacy keyword).

Reply:
How do I make it a subsection? In the rest file there are no level headings and they just render when I build the file.